google-api-grpc/proto-google-cloud-monitoring-v3/src/main/java/google/monitoring/v3/DroppedLabelsOuterClass.java

```diff
@@ -1,914 +0,0 @@
-// Generated by the protocol buffer compiler.  DO NOT EDIT!
```

Comment:
What this java file supposed to be deleted?

Reply:
That file was erroneously generated a long time ago. It's missing `com.` in the package. The real version is https://github.com/googleapis/google-cloud-java/blob/master/google-api-grpc/proto-google-cloud-monitoring-v3/src/main/java/com/google/monitoring/v3/DroppedLabelsOuterClass.java